src/hdr_histogram.h

```diff
@@ -33,8 +33,36 @@
     using std::memory_order_seq_cst;
 
     using std::atomic_int_least64_t;
-#else /* for std C11 compiler */
-    #include <stdatomic.h>
+#else /* for std C compiler */
+    #ifdef _MSC_VER /* for MSVC as it dosn't support stdatomic.h */
+        /*
+         * MSVC dose not support stdatomic.h  so we have to use a volatile
+         * variable and Interlock* methods
+         *
+         * MSVC treats volatile variables difrently by defualt and automaticly
+         * provids higher memory ordering than an ISO compliant volatile.
+         *
+         * See link bellow for more information.
+         * https://docs.microsoft.com/en-us/cpp/cpp/volatile-cpp?view=vs-2017
+         */
+        #define WIN32_LEAN_AND_MEAN
+        #include <Windows.h>
+
+        typedef volatile int64_t atomic_int_least64_t;
+        #define memory_order_relaxed 0
+
+        #define atomic_load_explicit(loadAdd, memory_order) *loadAdd
```

Comment:
Suggest you add brackets around `*loadAdd` to ensure there's no presidence issues when using this.

Reply:
Done